shared/constants/network.ts

```diff
@@ -453,6 +456,8 @@ export const NUMBERS_MAINNET_IMAGE_URL = './images/numbers-mainnet.svg';
 export const NUMBERS_TOKEN_IMAGE_URL = './images/numbers-token.png';
 export const SEI_IMAGE_URL = './images/sei.svg';
 export const NEAR_IMAGE_URL = './images/near.svg';
+export const MATCHAIN_IMAGE_URL = './images/matchain.svg';
+export const APE_TESTNET_IMAGE_URL = './images/ape.svg';
```

Comment:
There is no need to add `APE_TESTNET_IMAGE_URL`. Can you please remove it?

Reply:
Done!